examples/server_hardware.py

```diff
@@ -82,6 +82,20 @@
 print("Added rack mount server '%s'.\n  uri = '%s'" %
       (server['name'], server['uri']))
 
+# Create Multiple rack-mount servers
+if oneview_client.api_version >= 600:
+    options_to_add_mult = {
+        "mpHostsAndRanges": config['server_mpHostsAndRanges'],
+        "username": config['server_username'],
+        "password": config['server_password'],
+        "licensingIntent": "OneView",
+        "configurationState": "Managed",
+        "initialScopeUris": ["/rest/scopes/7a7db914-ca1d-4112-9189-4ce1a4774ce3"]
```

Comment:
should likely avoid hard coding a scope uri into an example, I'd suggest adopting one of the following choices:
1- If this is not a **must** for the example to work, drop it
2- Try and perform a get_all on the scopes, get the first one and use it here if it exists, otherwise skip this part of the example
3- Adding in a 'scope_uris:' field at the beginning of the file, which can optionally be filled by the user

Reply:
Dropping this field as this is not mandatory, and can be filled by the user if required.